src/python/pants/backend/cc/dependency_inference/rules.py

```diff
@@ -12,17 +12,15 @@
 from pants.backend.cc.target_types import CCDependenciesField, CCSourceField
 from pants.build_graph.address import Address
 from pants.core.util_rules import stripped_source_files
-from pants.core.util_rules.stripped_source_files import StrippedFileName, StrippedFileNameRequest
-from pants.engine.fs import DigestContents
-from pants.engine.internals.native_engine import Digest
-from pants.engine.internals.selectors import Get, MultiGet
-from pants.engine.rules import Rule, collect_rules, rule
+from pants.core.util_rules.stripped_source_files import StrippedFileNameRequest, strip_file_name
+from pants.engine.internals.graph import determine_explicitly_provided_dependencies, hydrate_sources
+from pants.engine.internals.selectors import concurrently
```

Comment:
isn't `concurrently` meant to be a public plugin api function?

Reply:
Hmm, that's weird, it was re-exported from engine rules as:

`from pants.engine.internals.selectors import concurrently as concurrently  # noqa: F401`

Ah, I see - `MultiGet` to `concurrently` was just a find/replace, so it was incorrectly imported in the first place. Will fix. Good catch, I didn't even look at that.